### PR TITLE
describe migration scenario when going to a new server

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

File: doc/integrator/install_application.rst [hunk 1]
```diff
@@ -355,3 +355,30 @@ Your application should be available at:
...
+
+If you are migrating to a new server, keep in mind that your variable
+``VISIBLE_WEB_HOST`` must contain the exact host name that browsers should use
+to access your site. Consider the following migration scenario:
+your current site runs on server ``old-site.customer.ch`` with the visible host name
+``gis.customer.ch``. You wish to setup a new server ``new-site.customer.ch``,
+install the application and test it, and then switch your DNS so that
+``gis.customer.ch`` now points to ``new-site.customer.ch``.
+To accomplish this, you must proceed as follows:
+
```
Comment (marionb) [inline doc/integrator/install_application.rst]: To be clearer of where your instructions part you could add:

   
     * install the application on ``new-site.customer.ch``


as a first stepp

Comment (jwkaltz) [reply doc/integrator/install_application.rst]: Better now?

Comment (marionb) [reply doc/integrator/install_application.rst]: LGTM

File: doc/integrator/install_application.rst [hunk 2]
```diff
@@ -355,3 +355,31 @@ Your application should be available at:
 
 Where the ``<hostname>`` is directly linked to the virtual host,
 and the ``<instanceid>`` is the value you provided before.
+
+.. _integrator_install_application_migrate_server:
```
Comment (sbrunner) [inline doc/integrator/install_application.rst]: Why this?

Comment (jwkaltz) [reply doc/integrator/install_application.rst]: I guess I thought I was going to create a reference to this section in another place. But finally, there isn't any reference to this, and I think we don't need one. Shall I remove it?

Comment (sbrunner) [reply doc/integrator/install_application.rst]: I think that will be cleaner :-)

Comment (jwkaltz) [reply doc/integrator/install_application.rst]: Done in https://github.com/camptocamp/c2cgeoportal/pull/3536

**Pull request merged**